uiowa/uiowa
# Feature: AoS sidebar information block

Resolves https://github.com/uiowa/uiowa/issues/2967. 

# How to test

- `blt ds --site=admissions.uiowa.edu`
- `yarn workspace admissions_core build`
- `drush @admissions.local cr`
- Go to http://admissions.local.drupal.uiowa.edu/areas-of-study/accounting and verify that sidebar block is showing up with a gray background and custom heading
<img width="299" alt="Screen Shot 2021-02-01 at 4 13 37 PM" src="https://user-images.githubusercontent.com/1036433/106524760-a3c7dd80-64a8-11eb-8be2-eef924f0ed70.png">
